discord-scripts/invite-management.ts

```diff
@@ -0,0 +1,181 @@
+import { Robot } from "hubot"
+import {
+  Client,
+  TextChannel,
+  VoiceChannel,
+  StageChannel,
+  ApplicationCommandOptionType,
+} from "discord.js"
+
+async function createInvite(
+  discordClient: Client,
+  channelId: string,
+  maxAge: number | string = 86400,
```

Comment:
Let's add relevant constants to globals.ts (I did this already @ https://github.com/thesis/valkyrie/pull/271/files#diff-75c9d9a77a387c5f245414547310fba482857ff1d90b45e0a43381a3ecd25e32 but that PR is a mess so we can't just cherry-pick a commit), and then define this in terms of those constants (e.g. `1 * WEEK`).